### ISSUE TITLE
Include tech data from other countries than Europe

### ISSUE THREAD

**Issue by lisazeyen** [PyPSA/technology-data#63]
## Include technology assumptions from other countries than Europe

Investment costs vary depending on the country. Currently the cost assumptions are mainly based on cost assumptions for Europe. It would be nice to extend the technology-data also to other countries.

@pz-max @euronion do you have some insights for cost assumptions in other countries? 
@tomkourou kindly offered to help to feed data from other countries back into technology-data

**Comment by pz-max** [comment]
@lisazeyen we are still using the EU default for Africa. Investment costs could vary according to [interest rates](https://tradingeconomics.com/country-list/interest-rate) for short-term considerations. 

For long-term planning, I think a uniform cost assumption is fair enough, otherwise, you might introduce a bias. The current interest rate in Argentina is 52% assuming this value for 2050 is probably not useful.

**Comment by martavp** [comment]
Indeed @pz-max https://www.nature.com/articles/s41467-019-12468-z
( I linked the paper not because I support country-specific discount rates but because I think it nicely shows the large impact of this assumption)

**Comment by Tomkourou** [comment]
I agree with @martavp and @pz-max that the interest rate is an important factor and should be country specific. I would go one step further and point out that both CAPEX and OPEX of generation technologies can vary quite a lot country to country. This is especially true when talking about developing countries and technologies with high learning rates like solar and wind. But also true for nuclear in certain regions like Russia and China for example. From a political perspective, making long term decarbonisation studies using global values is often criticised and not taken as credible. Its important where we can to use local estimates and fill the rest with global values (for prospective technologies like electrolysers for example.) At Agora we have already collected a lot of country specific cost data for the ASEAN region in particular and would be happy to contribute :)
